docs/uptime-guide/install.asciidoc

```diff
@@ -51,7 +51,7 @@ Uptime requires the setup of monitors in Heartbeat.
 These monitors provide the data you'll be visualizing in the {kibana-ref}/xpack-uptime.html[Uptime app].
 
 For instructions on installing and configuring Heartbeat, see the *Setup Instructions* in {kib}.
-Additional information is available in {heartbeat-ref}/heartbeat-configuration.html[Configure Heartbeat].
+Additional information is available in {heartbeat-ref}/configuring-howto-heartbeat.html[Configure Heartbeat].
```

Comment:
I think the quickstart guide link makes more sense here

Reply:
Agreed. I was doing a search/replace, but will change this as recommended.

Reply:
(Note that the file moved, so this change is in https://github.com/elastic/observability-docs/pull/41)